mgmtfn/k8splugin/contivk8s/k8s_cni_test.go

```diff
@@ -162,6 +164,17 @@ func setupTestServer() {
 		logger.Infof("k8s test plugin closing %s", driverPath)
 	}()
 
+	// make sure the listener is ready before returning
+	for count := 0; count < 5; count++ {
```

Comment:
Because this is a one time thing and we are unlikely to hit the "sleep" part of it, I am not sure of the value it brings. But, I agree that using chan will help the wait to be optimized.